robosherlock/src/core/src/scene_cas.cpp

```diff
@@ -112,4 +114,59 @@ void SceneCas::setFS(const char *name, const uima::FeatureStructure &fs)
   view->setSofaDataArray(fs, UnicodeString::fromUTF8(mime));
 }
 
+std::string SceneCas::getCAStoString(uima::CAS &tcas, bool saveWithPointCloud)
```

Comment:
Could you please add a simple test for this method?
It can be called on a very simple CAS and should just check for the presence of a value of your choice. 
The most important thing is just to test the basic functionality.

Reply:
where do u want the test to be?

Reply:
Here: https://github.com/RoboSherlock/robosherlock/tree/master/robosherlock/test/robosherlock 😃

Reply:
@Sanic added a very small test with the basic cas. Hope that is enough.